set-start.js

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env node
+
+var path = require('path')
+var fs = require('fs')
+
+// use process.env to access npm config environment variables
+var saveRequested = process.env.npm_config_save
+// https://docs.npmjs.com/misc/config#environment-variables
+// https://docs.npmjs.com/misc/config#save
```

Comment:
No problem!

And yes, I'm still chugging along on your reviews and comments (thank you guys by the way!). I meant to reply to each last night but ended up stumbling across some interesting npm script stuff I'd never read before and fell down a rabbit hole of documentation and stackoverflow.